src/app/components/cds/slideout/Slideout.module.css

```diff
@@ -0,0 +1,87 @@
+.slideoutMain {
+  box-shadow: -5px 0 5px 2px rgb(0 0 0 / .3);
+  height: 100%;
+  max-width: 600px;
+  min-width: 320px;
+  position: fixed;
+  top: 0;
+  width: 25vw;
+}
+
+.slideoutContainer {
+  background-color: var(--otherWhite);
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+  position: relative;
+}
+
+.slideoutTitle {
+  background-color: var(--brandLight);
+
+  .slideoutTitleContent {
+    align-items: center;
+    display: flex;
+    flex-wrap: nowrap;
+    gap: 8px;
+    justify-content: space-between;
+    min-height: 80px;
+    padding: 10px;
+
+    h6 {
+      margin: 0;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  }
+}
+
+.slideoutBody {
```

Comment:
you could add `flex: 1 1 auto;` to get this to expand to fill the remaining space

Reply:
Updated, content box no longer shrinks with small amounts of content